README.md

```diff
@@ -1 +1,44 @@
-# storefront
\ No newline at end of file
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app), using the [Redux](https://redux.js.org/) and [Redux Toolkit](https://redux-toolkit.js.org/) template.
+
```

Comment:
Add more detail to your readme. For example a link to your PR, deployment link, a description of the lab, and/or a UML. You can also refers to the README templete

Reply:
README.md done